src/fluree/db/json_ld/reify.cljc

```diff
@@ -414,8 +414,8 @@
 
 (defn trace-commits
   "Returns a list of two-tuples each containing [commit proof] as applicable.
-  First commit will be t value of '1' and increment from there."
-  [conn latest-commit]
+  First commit will be t value of `to-t` and increment from there."
+  [conn latest-commit to-t]
```

Comment:
Minor, but I think this argument should be called `from-t`. We use `from-t` in other places when we are talking about the lowest t-value in a range, so mixing the usage here could lead to confusion, even though this function is tracing the commits backwards.